### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Vishal30M**
In this project I did the following:

- I set up Linters in my Repository.
- I modified the README.md file.
- Added HTML and CSS files.
- I fixed the linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+   
+
+<div class="about-me">
+    <h3 class="About">About <br> Myself</h3>
+    <p class="me">Hello I’m a software developer! I can help you build a product , feature or website Look through
+        some of my work and experience! If you like what you see and have a project you need coded, don’t
+        hestiate to contact me.</p>
+    <div class="social-media">
+        <a href="#" class="contact">LET'S CONNECT </a>
```
Comment (Lucas-Masaba) [inline index.html:175]: - [ ] It would be better if you make this a `hx` tag instead of an `a` tag since it is a heading 🏄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+                      <img src="/image/EllipseHTML.png" alt="" /> <p>HTML</p>
+                    </a>
+                  </li>
+               
+                  <li>
+                    <a href="#">
+                      <img src="/image/EllipseCSS.png" alt="" /> <p>JavaScript</p>
+                    </a>
+                  </li>
+                </ul>
```
Comment (omarsalem7) [inline index.html:226]: - [ ] As requested by the previous reviewer use `ul and li` to make a language list but after you made it there issue in your design that your design does not match Figma.
- Your design
image
- Figma
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+        <section class="content">
+            <div class="introduction">
+                <h1 class="intro">I’m Vishal <br> Glad to see <br> you! </h1>
+                <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate tocontact me.</p>
+            </div>
+    
+            <div class="social-media">
+                <a href="#" class="contact">LET'S CONNECT </a>
```
Comment (omarsalem7) [inline index.html:38]: - [ ] As mentioned by the previous reviewer, please use `hx` instead of `<a>` because it is a heading and to improve SEO.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Vishal30M, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

you-are-near

 ## Highlights
✔️ All linter checks pass
✔️ Descriptive pull request

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+                  </li>
+               
+                  <li>
+                    <a href="#">
+                      <img src="/image/EllipseCSS.png" alt="" /> <p>JavaScript</p>
+                    </a>
+                  </li>
+                </ul>
+                  <div class="framework">
+                    <h3 >Framework</h3>
```
Comment (algerina) [inline index.html:228]: - [ Optional] You did well but there's a typo here it should be `Framworks`  instead of `Freamwork`.

**Review by algerina — APPROVED** [review]
Hi @Vishal30M , 
Good job so far! 

May I highlight the following:

- The repo looks professional. :heavy_check_mark: 
- There are no linter errors :heavy_check_mark: 
- You have descriptive :heavy_check_mark: 
- You implemented the changes required by the previous reviewers :heavy_check_mark: 

## STATUS: APPROVED :muscle::clap:


Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**